schema/schemaspec/schemahcl/hcl.go

```diff
@@ -17,12 +17,24 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
+// Unmarshal parses the Atlas HCL-encoded data and stores the result in the target.
+func Unmarshal(body []byte, target interface{}) error {
```

Comment:
```suggestion
func Unmarshal(data []byte, v interface{}) error {
```

https://pkg.go.dev/encoding/json#Unmarshal
https://pkg.go.dev/encoding/xml#Unmarshal